.ci/containers/membership-checker/main.go

```diff
@@ -91,12 +91,13 @@ func main() {
 			}
 		} else {
 			addAwaitingApprovalLabel(prNumber, GITHUB_TOKEN)
+			postAwaitingApprovalBuildLink(prNumber, GITHUB_TOKEN, projectId, commitSha)
 		}
 	}
 
 	// in community-checker job:
 	// remove awaiting-approval label from external contributor PRs
-	if target == "needs_approval" && !trusted {
```

Comment:
This change is to prevent labels not being removed for PRs whose contributor joins the org after creating the PR (e.g. https://github.com/GoogleCloudPlatform/magic-modules/pull/7610)